JSON/WithSensorData/SensorDataExample15.jsonld

```diff
@@ -0,0 +1,114 @@
+{
+  "@context": {
+    "epcis": "https://ns.gs1.org/epcis/",
+    "gs1": "https://gs1.org/voc/",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "vtype": "urn:epcglobal:epcis:vtype:",
+    "example-mda": "http://epcis.example.com/mda/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "example2": "http://epcis.example.com/ns/",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "cbv": "https://ns.gs1.org/cbv/",
+    "md": "https://ns.gs1.org/epcismd/",
+    "dcterms": "http://purl.org/dc/terms/",
+    "example1": "http://ns.example.com/epcis/",
+    "cbvmda": "urn:epcglobal:cbv:mda:"
+  },
+  "@graph": [
+    {
+      "@id": "ni:///sha-256;77c5b495d391767b1d3b8e7c669c9fee3965a778a9bfe793e85602c971b98436?ver=CBV2.0",
```

Comment:
Start from an existing JSON example and use `id, isA, action` etc instead of `@id, @type, epcis:action` etc